### PR TITLE
Update dependencies and perform a little cleanup

### PR DESCRIPTION
**Pull request by abelgardep**
Update some dependencies and perform a little cleanup over build gradles.

Update Android Studio version to 4.1

Needs https://github.com/owncloud/android-library/pull/351

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -8,25 +8,24 @@ buildscript {
...
-        appCompat = "1.1.0"
+        appCompat = "1.2.0"
         sqliteVersion = "2.1.0"
         lifecycleLiveData = "2.2.0"
         biometricVersion="1.0.1"
 
         // Kotlin
-        kotlinVersion = "1.3.72"
-        coroutinesVersion = "1.3.2"
+        kotlinVersion = "1.4.10"
```
Comment (hannesa2) [inline build.gradle:17]: I observed this causes the already commited 636ec200ac61a408 code style change

File: owncloudApp/build.gradle [hunk 2]
```diff
@@ -115,12 +102,6 @@ android {
 
         multiDexEnabled true
 
-        javaCompileOptions {
-            annotationProcessorOptions {
-                arguments = ["room.schemaLocation": "$projectDir/schemas".toString()]
```
Comment (hannesa2) [inline owncloudApp/build.gradle:120]: I'm not sure if this is smart. 
Room needs it for migration tests and without it, you will probably see a warning during build concerning this

Comment (abelgardep) [reply owncloudApp/build.gradle:120]: Yes, i know room needs it for migration tests. That's why it is included in the ownCloudData module build gradle. https://github.com/owncloud/android/blob/master/owncloudData/build.gradle#L25

File: build.gradle (hunk 1 above)
Comment (theScrabi) [reply build.gradle:17]: What does that mean?

**Review by theScrabi — APPROVED** [review]

**Comment by jesmrec** [comment]
No much to test here, config stuff. App builds, tests pass. Go ahead with this.

**Pull request merged**